assignments/callbacks.js

```diff
@@ -27,23 +27,29 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
```

Comment:
You are not invoking your functions anywhere. You could want to write a separate logger function and invoke the various functions passing that as the second argument.

Reply:
@GianBarGian Fixed, I think :)

Reply:
No, the way the functions was declared was fine. 
It's just that you are not trying to use it anywhere after is declared.  
In this way you are not trying to actually use callbacks functions. You could want to maybe write a logger function and pass it as the second argument of your functions.
If it is not clear DM me.

Reply:
Ooooh ok, I get it now! Fixed (now for real haha).

Reply:
Yeah now it looks perfect. Well done!